cmd/krel/cmd/push.go

```diff
@@ -147,5 +153,89 @@ func init() {
 
 func runPushBuild(opts *pushBuildOptions) error {
 	log.Println("unimplemented")
+
+	var latest string
+	releaseKind := opts.releaseKind
+
+	// Check if latest build uses bazel
+	dir, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	isBazel, err := release.BuiltWithBazel(dir, releaseKind)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if isBazel {
+		log.Println("Using Bazel build version.")
```

Comment:
Its a nitty nit, but we could remove the dot at the end of a log to have a uniform logging layout. :)